web-components/src/global.css.js

```diff
@@ -251,43 +252,50 @@ merch-card [slot='heading-xl'] {
     color: var(--merch-color-grey-80);
 }
 
-merch-card [slot='callout-text'] {
-    display: inline-block;
+merch-card [slot='callout-content'] {
+    display: flex;
```

Comment:
what's the rationale behind all those display: flex just for the sake of adding spaces?

Reply:
We will require one slot div for entire callout content. 
Then a div to separate the individual callouts. We don't want the background across entire callouts. This div will have texts, links, inline pricing etc inside another div and optionally an image tag.
I had to wrap the above div inside an additional div to place the image on top right. Note that for single line callout text, the image comes at last inline, but for multiline I had to use flex to push the image on right, keeping the text div as inline block.